src/components/common/markets/markets-section.tsx

```
@@ -36,26 +36,158 @@ export default function MarketsSection({
+                        <Box
+                          key={`skeleton-grid-${cycleIndex}-${index}`}
+                          flex='1 1 calc(50% - 8px)'
+                          minW='calc(50% - 8px)'
+                        >
+                          <Skeleton height={164} />
+                        </Box>
+                      ))}
+                    </Flex>
+                  </React.Fragment>
+                ))}
+              </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)